RELEASE/scripts/autoscend/auto_providers.ash

```diff
@@ -1212,3 +1212,252 @@ boolean provideMoxie(int amt, boolean doEquips)
+		if(target != $familiar[none] && target != my_familiar())
+		{
+			int famWeight = familiar_weight(target) + weight_adjustment();
+			delta += numeric_modifier(target, "Meat Drop",famWeight,$item[none]);
+			auto_log_debug("With using familiar: " + target + " we can get to " + result());
+		}
+		else
+		{
+			auto_log_debug("Already have desired familar, " + target + ", active.");
+		}
+		if(pass())
+			return result();
+	}
+	void handleEffect(effect eff)
+	{
+		if(speculative)
+		{
+			delta += numeric_modifier(eff, "Meat Drop");
+		}
+		auto_log_debug("We " + (speculative ? "can gain" : "just gained") + " " + eff.to_string() + ", now we have " + result());
+	}
+	boolean tryEffects(boolean [effect] effects)
+	{
+		foreach eff in effects
+		{
+			if(buffMaintain(eff, 0, 1, 1, speculative))
+				handleEffect(eff);
+			if(pass())
+				return true;
+		}
+		return false;
+	}
+	if((get_property("sidequestArenaCompleted") == "fratboy") && !get_property("concertVisited").to_boolean() && (have_effect($effect[Winklered]) == 0))
+	{
+		if(is_professor())
+		{
+			//Need to manually equip because professor
+			if(!have_equipped($item[beer helmet])) equip($item[beer helmet]);
+			if(!have_equipped($item[distressed denim pants])) equip($item[distressed denim pants]);
+			if(!have_equipped($item[bejeweled pledge pin])) equip($item[bejeweled pledge pin]);
+		}
+		else
+		{
+			outfit("Frat Warrior Fatigues");
+		}
+		cli_execute("concert 2");
+	}
+	handleBjornify($familiar[Hobo Monkey]);
+	// unlimited skills
+	if(tryEffects($effects[
+		Polka of Plenty,
+		Disco Leer
+	]))
+		return result();
+	if(canAsdonBuff($effect[Driving Observantly]))
+	{
+		if(!speculative)
+			asdonBuff($effect[Driving Observantly]);
+		handleEffect($effect[Driving Observantly]);
+	}
+	if(pass())
+		return result();
+	if(bat_formWolf(speculative))
+	{
+		handleEffect($effect[Wolf Form]);
+	}
+	if(pass())
+		return result();
+	if(auto_birdModifier("Meat Drop") > 0)
+	{
+		if(tryEffects($effects[Blessing of the Bird]))
+			return result();
+	}
+	if(auto_favoriteBirdModifier("Meat Drop") > 0)
+	{
+		if(tryEffects($effects[Blessing of Your Favorite Bird]))
+			return result();
+	}
+	if(isActuallyEd())
+	{
+		if(!have_skill($skill[Gift of the Maid]) && ($servant[Maid].experience >= 441))
+		{
+			visit_url("charsheet.php");
+			if(have_skill($skill[Gift of the Maid]))
+			{
+				auto_log_warning("Gift of the Maid not properly detected until charsheet refresh.", "red");
+			}
+		}
+		if(tryEffects($effects[
+		Purr of the Feline
+		]))
+		return result();
+	}
+	songboomSetting("meat");
+	// items
+	if(tryEffects($effects[
+		Greedy Resolve,
+		Kindly Resolve,
+		Heightened Senses,
+		Big Meat Big Prizes,
+		Human-Machine Hybrid,
+		Human-Constellation Hybrid,
+		Human-Humanoid Hybrid,
+		Human-Fish Hybrid,
+		Cranberry Cordiality,
+		Patent Avarice,
+		Car-Charged,
+		Heart of Pink,
+		Sweet Heart,
+		Earning Interest,
+		Bet Your Autumn Dollar,
+		Flapper Dancin\',
+		shadow waters

```

Comment:
I'm not sure shadow waters belongs here. I'd propose taking it out of here and keeping stand alone in the 8 bit realm handler as waters are so good there. Another option would be to take it out of here still, but then only try to get it if `useEveryting` is true

Reply:
I think waters to doEverything is the move because it is so useful for here. We can also add a direct call in 8 bit realm to try to get it, but because we bounce in and out of the 8 bit realm with the habitated monsters, I think it will be wasted